client/ayon_core/tools/common_models/hierarchy.py

```diff
@@ -272,6 +279,48 @@ def get_folder_items(self, project_name, sender):
             self._refresh_folders_cache(project_name, sender)
         return self._folders_items[project_name].get_data()
 
+    def get_assigned_folder_paths(
+        self, project_name, sender=None, assignee=None
+    ):
+        """Get folder paths assigned to user by project name.
+
+        If a child folder is assigned (i.e. /season/seq/sh010) this
+        function will also return all the parent entities on the
+        hierarchy (i.e. ["/season", "/season/seq"]) as those are
+        implicitly assigned to the user as well.
```

Comment:
I understand reasoning as to why we might want to include parents for filtering for filtering logic where we still want to access the child. But the wording here is a bit weird since I don't understand why "implicitly" it means the parent folder is also assigned to the user - that seems odd and not the case in the frontend?

Reply:
what do you suggest?

Reply:
I'd suggest just explaining it as:

````
        If a child folder is assigned (i.e. /season/seq/sh010) this
        function will also return all the parent entities on the
        hierarchy (i.e. ["/season", "/season/seq"]) because it
        contains children folders assigned to the user.
```

But maybe I'm misinterpreting why it's returning the parent entities?

Reply:
I understand both the same way but I will change it to your phrasing